bindings/python/test/estimator/test_od_least_squares_solver.py

```
@@ -182,6 +182,114 @@ def test_properties(
         assert isinstance(analysis.solver_analysis, LeastSquaresSolver.Analysis)
 
 
+@pytest.fixture
+def rms_error() -> float:
```

Ruff (0.8.2)

Redefinition of unused `rms_error` from line 78

(F811)